src/CommunityToolkit.Maui.Core/Capabilities/StatusBar/StatusBar.ios.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Platform;
+
+namespace CommunityToolkit.Maui.Core.Capabilities;
+
+static partial class StatusBar
+{
+	static void PlatformSetColor(Color color)
+	{
+		var uiColor = color.ToPlatform();
+
+		if (PlatformVersion.IsiOS13OrNewer)
+		{
+			const int statusBarTag = 38482;
```

Comment:
I've never used `window.ViewWithTag()` before.

Where is tag #38482 set? Is that set in OS by Apple or in .NET MAUI? Any fears of this number changing in the future by Apple or .NET MAUI?

Reply:
Still looking for this magic number, but didn't find the explanation yet... But looks like the people on [swift world use](https://mobikul.com/get-status-bar-view-in-ios-13-and-above/) this number as well...
Maybe @KSemenenko knows the history behind this magic number...

Reply:
It works by adding a box or something like that.
And this id everyone uses as a marker that this is the color of the status bar :)